src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -117,6 +117,8 @@ public static bool IsPrivilegedProcess
 #endif
 
         public static bool IsThreadingSupported => !IsBrowser && !IsWasi;
```

Comment:
This could be used instead of introducing a new property.

Reply:
But it's not true, right? Threading is supported in Browser and Wasi and supported != switched on.

Reply:
I guess `IsThreadingSupported` will be removed soon anyway. For now a lot of tests are blocked by it but when the threading will be ready, the property probably will get trashed and they will be unblocked.